plugins/vst_base/RemoteVstPlugin/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ FOREACH( OUTPUTCONFIG ${CMAKE_CONFIGURATION_TYPES} )
 	SET("CMAKE_RUNTIME_OUTPUT_DIRECTORY_${OUTPUTCONFIG}" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")
 ENDFOREACH()
 
+ADD_SUBDIRECTORY(../../../src/common common)
```

Comment:
Any advantages of using relative path rather than `LMMS_SOURCE_DIR`?

Reply:
I think `LMMS_SOURCE_DIR` is better. Fixed here and in `plugins/zynaddsubfx/CMakeLists.txt`.